### DISCUSSION TITLE
为什么不在服务器列表添加多选操作？

### DISCUSSION THREAD

**Discussion by ZhangYiQiu** [isontheline/pro.webssh.net#1022] [Ideas]
如果可以多选，我就可以同时删除多条记录了。

并且，目前服务器列表也不支持左滑删除，如果我要删除多项记录，我只能长按，选择删除，然后再下一条记录上重复这个操作。

因为新版本新增了“全部”类型，我想对我现有的ssh或者sftp服务器的类型批量设定，为它们同时勾选上ssh和sftp，但是目前没有这种批量操作，我只能一个个修改，然后再删除多余的记录。

顺便问一句，如果我想更正部分翻译，我应该怎么做？

**Comment by isontheline** [comment]
> If I can choose more, I can delete multiple records at the same time.
> Moreover, at present, the server list does not support left-swipe deletion. If I want to delete multiple records, I can only long press, choose to delete, and then repeat this operation on the next record.
> Because the new version has added "all" types, I want to set the types of my existing ssh or sftp servers in batches and check ssh and sftp for them at the same time, but there is no such batch operation at present. I can only modify them one by one and then delete the redundant records.

Right now batch select is not available but you can use "servedit" in order to update or delete in batch inside mashREPL :
https://webssh.net/documentation/help/howtos/mashREPL/servedit/

> By the way, what should I do if I want to correct part of the translation?

Translation file is not yet available, I will push it on this repo soon.
By the way which translation sentence is wrong?

**Reply by ZhangYiQiu** [reply]
![image](https://github.com/isontheline/pro.webssh.net/assets/22910338/6c5c15be-5334-458f-afc0-a5409ed72ee6)

1 应更改为 导入
2 应更改为 生成


目前我发现这两个翻译不准确，如果我之后发现了新的，我会补充

**Comment by isontheline** [comment]
Thank you @ZhangYiQiu 
Translations will be fixed inside the next upcoming beta release
